tic-tac-toe/src/app/screens/About/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+
+class About extends Component {
+  componentDidMount() {
+    if (!localStorage.getItem('token')) {
+      this.props.dispatch(push('/'));
+    }
+  }
+
+  render() {
+    return (
+      <Fragment>
+        <h1 className="titel-game">History TIC-TAC-TOE</h1>
+        <p className="text-game">
+          <img
+            className="img-game"
+            alt="imagen-tic-tac-toe"
+            src={'https://thumbs.gfycat.com/PoisedGrippingFox-size_restricted.gif'}
+          />
+          Games played on three-in-a-row boards can be traced back to ancient Egypt,[5] where such game boards
```

Comment:
Extract this text to a constant and create a file strings in this folder and put it.